lib/machine/fingerprintSupport.ts

```diff
@@ -101,16 +100,20 @@ export interface FingerprintImpactHandlerConfig {
 export type RegisterFingerprintImpactHandler = (sdm: SoftwareDeliveryMachine, registrations: Aspect[]) => FingerprintHandler;
 
 export const DefaultTargetDiffHandler: FingerprintDiffHandler =
-    async (ctx, diff, aspect) => {
-        const v: Vote = await checkFingerprintTarget(
-            ctx.context,
-            diff,
-            aspect,
-            async () => {
-                return diff.targets;
-            },
-        );
-        return v;
+    async (ctx, diffs, aspect) => {
+        const abs = _.filter(diffs, diff => !diff.from);
+        return [
+            ... await Promise.all(
+                    _.map (
+                        _.difference(diffs, abs), diff => checkFingerprintTarget(
+                            ctx.context,
+                            diff,
+                            aspect,
+                            async () => diff.targets ))),
+            ... _.map(abs, () => {
+                return {abstain: true};
```

Comment:
Format

Reply:
@cdupuis Thanks, but this seems like a job for a computer to me. Do we not have an auto-formatter available?